docs/using-kontena/nodes.md

```diff
@@ -42,6 +42,36 @@ The re-scheduling of grid services will happen after the node offline grace peri
 - `ha`: 2 minutes
 - `random`: 30 seconds
 
+### Node availability
+
+Node availablity for scheduling services can be controlled with `kontena node availablity` command. The current supported states are: `drain`and `active`.
```

Comment:
Would `kontena node update --availability=active moby` also work? Not saying which is better.